cves/CVE-2017-16840.yml

```diff
@@ -64,7 +64,13 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  When processing video media, non-Haar wavelets are used to transform 
```

Comment:
What are Haar wavelets?

Reply:
I also wondered this. Please explain this domain-specific lingo.